### PR TITLE
[new release] zarith-ppx (0.1)

### PR DESCRIPTION
**Pull request by Drup**
Literals for Zarith's arbitrary-precision integers and rationals

- Project page: <a href="https://github.com/Drup/Zarith-ppx">https://github.com/Drup/Zarith-ppx</a>

##### CHANGES:

Hello.

### CODE REVIEW

**Comment by mseri** [comment]
Osx fails wi5h ```
#=== ERROR while compiling zarith-ppx.0.1 =====================================#
# context     2.0.3 | macos/x86_64 | ocaml-system.4.07.1 | file:///Users/travis/build/ocaml/opam-repository
# path        ~/.opam/ocaml-system/.opam-switch/build/zarith-ppx.0.1
# command     ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p zarith-ppx -j 1
# exit-code   1
# env-file    ~/.opam/log/zarith-ppx-55621-756954.env
# output-file ~/.opam/log/zarith-ppx-55621-756954.out
### output ###
#         diff alias test/runtest (exit 2)
# (cd _build/default/test && /usr/bin/diff -dEbZBt testppx.result testppx.expected)
# /usr/bin/diff: invalid option -- Z
# /usr/bin/diff: Try `/usr/bin/diff --help' for more information.
```

File: packages/zarith-ppx/zarith-ppx.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+name: "zarith-ppx"
+version: "0.1"
```
Comment (kit-ty-kate) [inline packages/zarith-ppx/zarith-ppx.0.1/opam]: not a blocker at all but those two fields are redundant with the information opam-repository already has

**Comment by Drup** [comment]
@mseri Oh dear, that diff seems not to be availble on mac... It's only used for test, should we disable tests on mac ? :/

**Comment by kit-ty-kate** [comment]
if there is no other reasonable option, this sounds ok

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**